Dockerfile

```
@@ -0,0 +1,50 @@
+FROM alpine:3.16
+
+# Config
+ARG TARGETARCH
+ARG PHP_VERSION=8.0.28-r0
```

```suggestion
ARG PHP_VERSION=8.0.29-r0
```
they changed version lately, in fact i was not able to build the image locally